packages/drawing/src/services/image-remote.service.ts

```diff
@@ -0,0 +1,111 @@
+/**
+ * Copyright 2023-present DreamNum Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import type { IImageRemoteService, IImageRemoteServiceParam, Nullable } from '@univerjs/core';
+import { ImageSourceType, ImageUploadStatusType, Tools } from '@univerjs/core';
+import type { Observable } from 'rxjs';
+import { Subject } from 'rxjs';
+
+export const ALLOW_IMAGE_LIST = ['image/png', 'image/jpeg', 'image/jpg', 'image/gif', 'image/bmp'];
+
+const ALLOW_IMAGE_SIZE = 5 * 1024 * 1024;
+
+export class ImageRemoteService implements IImageRemoteService {
+    private _waitCount = 0;
+
+    private _change$ = new Subject<number>();
+    change$ = this._change$ as Observable<number>;
+
+    setWaitCount(count: number) {
+        this._waitCount = count;
+        this._change$.next(count);
+    }
+
+    getWaitCount() {
+        return this._waitCount;
+    }
+
+    decreaseWaiting() {
```

Comment:
This method doesn't have to be public, since nobody calls it. And the method in the public should be removed as well.